### PR TITLE
l10n: Unify spelling

### PR DESCRIPTION
**Pull request by Valdnet**
Spelling unification in Nextcloud applications.

### CODE REVIEW

**Review by rakekniven — APPROVED** [review]

**Comment by Valdnet** [comment]
@daita Can it approve and merge?

**Review by rakekniven — CHANGES_REQUESTED** [review]
Please change "admin" to "administrator" as well.

Thank you.

**Comment by Valdnet** [comment]
Done 😉.

File: src/App.vue [hunk 1]
```diff
@@ -42,7 +42,7 @@
 						{{ t('social', '.well-known/webfinger isn\'t properly set up!') }}
 					</h3>
 					<p v-if="!serverData.checks.checks.wellknown">
-						{{ t('social', 'Social needs the .well-known automatic discovery to be properly set up. If Nextcloud is not installed in the root of the domain, it is often the case that Nextcloud can\'t configure this automatically. To use Social, the admin of this Nextcloud instance needs to manually configure the .well-known redirects: ') }}<a class="external_link"
+						{{ t('social', 'Social needs the .well-known automatic discovery to be properly set up. If Nextcloud is not installed in the root of the domain, it is often the case that Nextcloud cannot configure this automatically. To use Social, the admin of this Nextcloud instance needs to manually configure the .well-known redirects:') }} <a class="external_link"
```
Comment (rakekniven) [inline src/App.vue:45]: Another occurence of "admin" is here.

**Review by rakekniven — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 1 above)
Comment (Valdnet) [reply src/App.vue:45]: Oops. I wanted to do too quickly. Thanks.

**Comment by Valdnet** [comment]
Done 😉.

**Review by rakekniven — APPROVED** [review]

**Pull request merged**